File: seed/csharp-model/trace/src/SeedTraceClient/Submission/TracedTestCase.cs
using System.Text.Json.Serialization
using SeedTraceClient

namespace SeedTraceClient

public class TracedTestCase
{
    [JsonPropertyName("result")]
    public TestCaseResultWithStdout Result { get; init; }

    [JsonPropertyName("traceResponsesSize")]
    public int TraceResponsesSize { get; init; }
}

File: seed/rust-model/trace/src/traced_test_case.rs
use serde::{Deserialize, Serialize};

#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
pub struct TracedTestCase {
    pub result: TestCaseResultWithStdout,
    #[serde(rename = "traceResponsesSize")]
    pub trace_responses_size: i32,
}